### PR TITLE
Milestone 2: Finished the Portfolio-works section

### PR DESCRIPTION
**Pull request by prg-04**
## In this Milestone, I have done the following step to set up a " Portfolio-Works section"

- [ ] Use grid layout and flexbox to set up the works and about me section

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -55,6 +55,154 @@ <h2>I'm a Software Developer</h2>
...
     </section>
+    <section id="portfolio">
+      <div class="title">
+        <h2>My Recent Works</h2>
+        <hr />
+      </div>
+      <div class="plchdr">
+        <div class="img">
+          <img src="/images/plc-img.jpg" alt="" />
+        </div>
```
Comment (Owusu-Desmond) [inline index.html]: - [ ] Please you should not have the `alt` attribute empty for your images. You should add a descriptive `alt` attribute to your images. :pray:

File: index.html [hunk 2]
```diff
@@ -55,6 +55,154 @@ <h2>I'm a Software Developer</h2>
...
+            since the 1500s, when an unknown printer took a standard dummy text.
+          </p>
+          <ul>
+            <li><a href="">css</a></li>
+            <li><a href="">html</a></li>
+            <li><a href="">bootstrap</a></li>
+            <li><a href="">ruby</a></li>
+          </ul>
+          <button>See Project</button>
+        </div>
```
Comment (Owusu-Desmond) [inline index.html]: - [ ] Please make sure your see project button follows the button styling: Active, Hover, and Pressed to make it interactive. :pray:

 Screenshot from 2022-10-28 10-03-18

File: index.html [hunk 3]
```diff
@@ -55,6 +55,154 @@ <h2>I'm a Software Developer</h2>
...
+          A daily selection of privately personalized reads; no accounts or
+          sign-ups required. has been the industry's standard
+        </p>
+        <ul>
+          <li><a href="">html</a></li>
+          <li><a href="">bootstrap</a></li>
+          <li><a href="">Ruby</a></li>
+        </ul>
+        <button>See Project</button>
+      </div>
```
Comment (Owusu-Desmond) [inline index.html]: - [ ] Please make sure your see project button follows the button styling: Active, Hover, and Pressed to make it interactive. :pray:

 Screenshot from 2022-10-28 10-03-18

File: index.html [hunk 4]
```diff
@@ -55,6 +55,154 @@ <h2>I'm a Software Developer</h2>
...
```
Comment (Owusu-Desmond) [inline index.html]: - [ ] Please make sure your see project button follows the button styling: Active, Hover, and Pressed to make it interactive. :pray:

 Screenshot from 2022-10-28 10-03-18

File: index.html [hunk 5]
```diff
@@ -55,6 +55,154 @@ <h2>I'm a Software Developer</h2>
...
```
Comment (Owusu-Desmond) [inline index.html]: - [ ] Please make sure your see project button follows the button styling: Active, Hover, and Pressed to make it interactive. :pray:

 Screenshot from 2022-10-28 10-03-18

File: index.html [hunk 6]
```diff
@@ -55,6 +55,154 @@ <h2>I'm a Software Developer</h2>
...
```
Comment (Owusu-Desmond) [inline index.html]: - [ ] Please make sure your see project button follows the button styling: Active, Hover, and Pressed to make it interactive. :pray:

 Screenshot from 2022-10-28 10-03-18

**Review by Owusu-Desmond — CHANGES_REQUESTED** [review]
Hi @prg-04 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!



## Highlights
📌 No Linters error ✅
📌 Descriptive pull request✅
📌 Great working on the set up the project ✅
📌 Good commit messages ✅
📌 Professional README file ✅

## Required Changes ♻️
_Check the comments under the review._
- [ ] Please from the project requirements, you should stick to the design as much as possible. But as you can see you have some differences in the design. You can see the design below and compare it with your project design. :pray:

  | Your Design (With no margins in between the `about` and `works` sections and see project button width is more than the main `div`)|Figma Design (With margins and see project button width is not more than main `div`) |
  | :---:   | :---: | 
  | Screenshot from 2022-10-28 10-07-51 | Screenshot from 2022-10-28 10-07-04 |

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html [hunk 7]
```diff
@@ -55,6 +55,154 @@ <h2>I'm a Software Developer</h2>
...
       </div>
     </section>
+    <section id="portfolio">
+      <div class="title">
+        <h2>My Recent Works</h2>
+        <hr />
+      </div>
+      <div class="plchdr">
+        <div class="img">
+          <img src="/images/plc-img.jpg" alt="" />
```
Comment (mirouhml) [inline index.html]: - [ ] As the previous reviewer mentioned, please know that you should leave the alt attributes empty for your images. You should add a descriptive alt attribute to your images. 🙏

**Review by mirouhml — CHANGES_REQUESTED** [review]
# REQUIRED CHANGES 🔴

Hi @prg-04, 

Great work on making the changes requested by the previous reviewer 👏 

You've done well implementing some of the requested changes, but there are still some which haven't been addressed yet, but don't worry you're almost there!! 😉

image

 ## Highlights

- Great job setting up the correct linters ✔️
- Highly organized and clean code ✔️ 👨🏻‍💻
- Professional documentation ✔️
- Correct workflow ✔️
- Meaningful commit messages ✔️

 ## Required Changes ♻️
_Check the comments under the review._

- [ ] Please know that as the previous reviewer mentioned, you should stick to the design as much as possible. But as you can see you have some differences in the design:

  This is your design: 
  image
  And this is the design provided in the Figma template:
  image

- [ ] As the previous reviewer mentioned, please know that the project buttons should follow the button styling provided: Active, Hover, and Pressed to make it interactive:

  image


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by emmiiorji — APPROVED** [review]
Hi @prg-04, 

# Status: Approved ✅

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉

 
congratulations


## Good Highlights✔

- Linter checks are passing
- Good commit message
- Adherence to GitHub flow

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**